FeedStoreChallenge/Core Data/CoreDataFeed.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CoreDataFeed.swift
+//  FeedStoreChallenge
+//
+//  Created by OLEKSANDR ISAIEV on 13.05.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(CoreDataFeed)
+final class CoreDataFeed: NSManagedObject {
+	@NSManaged public var timestamp: Date
+	@NSManaged public var coreDataFeedImages: NSOrderedSet
+}
+
+extension CoreDataFeed {
+	var localFeed: [LocalFeedImage] {
+		return coreDataFeedImages
```

Comment:
To follow up the comment on `CoreDataFeed` please rename `coreDataFeedImages` to `images`

```suggestion
		return images
```